recipes/picojson/all/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+from conans import ConanFile, CMake, tools
+
+import os
+
+class PicoJSONTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        # Current dir is "test_package/build/<build_id>" and CMakeLists.txt is
+        # in "test_package"
+        cmake.configure()
+        self._is_multi_configuration = cmake.is_multi_configuration
```

Comment:
Although the `build()` and the `test()` are usually going to run together, setting this attribute here is weak. It will be better to use a property for this purpose so we make sure that methods can be run independently. Something like:

```
@propety
def _is_multi_configuration(self):
    cmake = CMake(self)
    return cmake.is_multi_configuration()
```

Reply:
Done: baecf503303fd01bf84c0ba53bdf9cc0877fbb1e